tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -446,3 +458,19 @@ type Tests(testOutputHelper) =
         test <@ singleBatchBackwards @ batchBackwardsAndAppend @ suboptimalExtraSlice @ singleBatchForward = capture.ExternalCalls @>
     }
 #endif
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_EVENTSTORE")>]
+    let ``Version is 0-based`` () = Async.RunSynchronously <| async {
+        let log, _ = output.CreateLoggerWithCapture()
+        let! connection = connectToLocalStore log
+
+        let batchSize = 3
+        let context = createContext connection batchSize
+        let id = $"{Guid.NewGuid():N}"
+        let decider = SimplestThing.resolve log context struct("SimplestThing", id)
+
+        let! before, after = decider.TransactEx(
+            (fun state -> state.Version, [SimplestThing.StuffHappened]),
+            mapResult = (fun s result -> s, result.Version))
+        test <@ before = 0L @>
```

Comment:
I tend to combine these with && so a fail will give more context

Reply:
I did `<@ [before; after] = [0L; 1L] @>` for terseness